[docs] [secure]Update YCQL page with password reset instructions for a superuser (#15677)
yugabyte/yugabyte-db: docs/content/preview/secure/enable-authentication/ycql.md

(3 rows)
```

#### Reset password for a superuser

> Superusers can alter the passwords for other users including the admin user. YCQL provides a feature to enable superusers to reset a lost/forgotten password using the[`ycql-allow-non-authenticated-password-reset`](../../../reference/configuration/yb-tserver/#ycql-allow-non-authenticated-password-reset) YB-TServer GFlag.

Reviewer: ```suggestion
Superusers can alter the passwords for other users including the admin user. If the [ycql_allow_non_authenticated_password_reset](../../../reference/configuration/yb-tserver/#ycql-allow-non-authenticated-password-reset) YB-TServer flag is set, superusers can also reset lost or forgotten passwords.
```

> Use the [`ycql-allow-non-authenticated-password-reset`](../../../reference/configuration/yb-tserver/#ycql-allow-non-authenticated-password-reset) YB-TServer GFlag in conjunction with the [`use_cassandra_authentication`](../../../reference/configuration/yb-tserver/#use-cassandra-authentication) YB-TServer GFlag to enable the password reset feature.

Reviewer 2: Please use underscore (`_`) instead of dash (`-`) as a separator. The flag name is `ycql_allow_non_authenticated_password_reset`.

> Use the [`ycql-allow-non-authenticated-password-reset`](../../../reference/configuration/yb-tserver/#ycql-allow-non-authenticated-password-reset) YB-TServer GFlag in conjunction with the [`use_cassandra_authentication`](../../../reference/configuration/yb-tserver/#use-cassandra-authentication) YB-TServer GFlag to enable the password reset feature.

Reviewer: ```suggestion
To enable the password reset feature, you must first set the [`use_cassandra_authentication`](../../../reference/configuration/yb-tserver/#use-cassandra-authentication) flag to false.
```

> The following example describes the steps to reset a password for the admin superuser created in the [Create a superuser](#create-a-superuser) step:

Reviewer: ```suggestion
For example, to reset the password for the admin superuser created in [Create a superuser](#create-a-superuser), do the following:
```
